[2024-05-08T19:51:47.049+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-08T19:51:47.067+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 7.3_filesensor.waiting_file scheduled__2022-07-31T00:00:00+00:00 [queued]>
[2024-05-08T19:51:47.073+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 7.3_filesensor.waiting_file scheduled__2022-07-31T00:00:00+00:00 [queued]>
[2024-05-08T19:51:47.074+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-08T19:51:47.086+0000] {taskinstance.py:2330} INFO - Executing <Task(FileSensor): waiting_file> on 2022-07-31 00:00:00+00:00
[2024-05-08T19:51:47.092+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2166) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-08T19:51:47.094+0000] {standard_task_runner.py:63} INFO - Started process 2167 to run task
[2024-05-08T19:51:47.094+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '7.3_filesensor', 'waiting_file', 'scheduled__2022-07-31T00:00:00+00:00', '--job-id', '701', '--raw', '--subdir', 'DAGS_FOLDER/07_FileSensor.py', '--cfg-path', '/tmp/tmpqiwnkce_']
[2024-05-08T19:51:47.095+0000] {standard_task_runner.py:91} INFO - Job 701: Subtask waiting_file
[2024-05-08T19:51:47.154+0000] {task_command.py:426} INFO - Running <TaskInstance: 7.3_filesensor.waiting_file scheduled__2022-07-31T00:00:00+00:00 [running]> on host ccc314eb571d
[2024-05-08T19:51:47.224+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='7.3_filesensor' AIRFLOW_CTX_TASK_ID='waiting_file' AIRFLOW_CTX_EXECUTION_DATE='2022-07-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-07-31T00:00:00+00:00'
[2024-05-08T19:51:47.225+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-08T19:51:47.234+0000] {baseoperator.py:399} WARNING - FileSensor.execute cannot be called outside TaskInstance!
[2024-05-08T19:51:47.242+0000] {base.py:84} INFO - Using connection ID 'fs_default' for task execution.
[2024-05-08T19:51:47.242+0000] {filesystem.py:85} INFO - Poking for file /tmp/file.txt
[2024-05-08T19:51:47.243+0000] {filesystem.py:89} INFO - Found File /tmp/file.txt last modified: 20240508195146
[2024-05-08T19:51:47.243+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-05-08T19:51:47.243+0000] {filesystem.py:85} INFO - Poking for file /tmp/file.txt
[2024-05-08T19:51:47.244+0000] {filesystem.py:89} INFO - Found File /tmp/file.txt last modified: 20240508195146
[2024-05-08T19:51:47.244+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-08T19:51:47.250+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=7.3_filesensor, task_id=waiting_file, run_id=scheduled__2022-07-31T00:00:00+00:00, execution_date=20220731T000000, start_date=20240508T195147, end_date=20240508T195147
[2024-05-08T19:51:47.267+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-08T19:51:47.292+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-08T19:51:47.298+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
